DirectProgramming/DPC++/ProjectTemplates/cmake-fpga/src/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 set(SOURCE_FILE main.cpp)
```

Comment:
I'm not looking into this file too much as we are going to remove this sample entirely for this release.
After you pointed me to it, I looked at it and realized it was:
1/ outdated
2/ in a place where FPGA users would not find it

Paul will be adding a replacement to this sample in DPC++FPGA/Tutorial/GettingStarted